### DISCUSSION TITLE
Standardize table status fields

### DISCUSSION THREAD

**Discussion by downdawn** [fastapi-practices/fastapi_best_architecture#137] [Ideas]
Standardize the use of the "status" field with an integer enumeration type for easier future expansion.

```
status: Mapped[int] = mapped_column(TINYINT(2), default=0, comment='状态（0停用 1正常）')
```
```
class StatusEnum(IntEnum):
    """状态"""

    enable = 1
    disable = 0
```

"is_active" can be deprecated.

**Comment by wu-clan** [comment]
Agree to implement.

**Comment by wu-clan** [comment]
Because the Mapped mapping is used, the type can be omitted: TINYINT

**Comment by wu-clan** [comment]
- https://github.com/fastapi-practices/fastapi_best_architecture/pull/143
